chapter_01/hinglish/introduction.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p><s>aur ek lagtar badhti hue community(samuday) jo isko behtar banane me liye kam karti hai</s></p><p>and isko istemal karne wali community kaafi badi hai. Agar apko koi bhi samasya aati hai to help karne k liye bahot log milenge.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/abhiabhi94/learn-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I have transformed this to:
`
Uske saath hi saath, ye puri tarike se free, aur open-source hai. Isko upyog karne wali community(samuday) kaafi badi hai. Agar apko koi bhi samasya aati hai to inn me se aapki help karne ke liye kafi log mil sakte hai.
`